tezos/interop/src/ffi.rs

```diff
@@ -310,13 +276,11 @@ pub fn call_protocol_rpc(
     request: ProtocolRpcRequest,
 ) -> Result<ProtocolRpcResponse, ProtocolRpcError> {
     runtime::execute(move |rt: &mut OCamlRuntime| {
-        ocaml_frame!(rt, (request_root), {
-            let request = to_ocaml!(rt, request, request_root);
-            let result = tezos_ffi::call_protocol_rpc(rt, request);
-            // TODO: should call_protocol_rpc be Result<_, OCamlErrorTrace> instead?
-            // looks like not, but verify and add a catch-all for unhandled cases
-            result.to_rust()
-        })
+        let request = request.to_boxroot(rt);
+        let result = tezos_ffi::call_protocol_rpc(rt, &request);
+        // TODO: should call_protocol_rpc be Result<_, OCamlErrorTrace> instead?
```

Comment:
This TODO is probably not relevant anymore, but I have to check.

Reply:
Regarding this comment: it is about `call_protocol_rpc` not being wrapped into the generic catch-all exception handler on the OCaml side, because it is not supposed to raise any exception. 

BUT I am are not really 100% sure it never raises an exception (unlike other unwrapped functions that one goes deeper and is more complicated), so I think I should wrap it the next time I make changes to the OCaml side (which will be soon), just in case. So I will leave this comment here for now, because I will have to update the code here when I change the OCaml side.